app/scripts/filters/resources.js

```diff
@@ -364,6 +364,18 @@ angular.module('openshiftConsole')
       return label;
     };
   })
+  .filter('routeLabelWithNoPath', function(routeHostFilter, routeWebURLFilter, isWebRouteFilter) {
```

Comment:
same thing as before, i would just add omitPath as a param to routeLabel as well and just pass it the parameter through